hevc-webrtc-07.nr

```diff
@@ -128,6 +128,9 @@ For the H.265 [H.265] codec, endpoints MUST support the payload formats
 defined in [RFC7798]. In addition, they MUST support Main Profile
 Level 3.1 (level-id=93) and SHOULD support Main Profile Level 4 (level-id=120).
 
+All NAL units included within an RTP payload MUST have the same TID value. An
```

Comment:
I think you should extend this to require same TID and LayerID within an RTP payload?

Reply:
Correct.